test/test_sparse_csr.py

```diff
@@ -25,7 +25,11 @@
 load_tests = load_tests
 
 # Unary Ufuncs supported by Sparse CSR Layout
-sparse_csr_unary_ufuncs = [op for op in sparse_csr_funcs if isinstance(op, UnaryUfuncInfo)]
+# since square is pow with exponent as 2, and pow has special handling
+# hence, square is temporarily filtered from the list, see: https://github.com/pytorch/pytorch/pull/69603#discussion_r793300941
+sparse_csr_unary_ufuncs = [
+    op for op in sparse_csr_funcs if isinstance(op, UnaryUfuncInfo) and op.name != "square"
```

Comment:
As an alternative to filtering here, we could set `supports_sparse_csr=False` in their OpInfo with a comment why they are marked as False (`since square is pow with exponent as 2, and pow has special handling...`).